geonode_mapstore_client/static/mapstore/configs/localConfig.json

```diff
@@ -320,10 +320,10 @@
                     "perms": [{ "type": "user", "value": "add_resource" }],
                     "items": [
                         {
-                            "labelId": "gnhome.layer",
```

Comment:
we should not change files inside `geonode_mapstore_client/static/mapstore/` because they are directly compiled from the script `npm run compile`